### PR TITLE
Kevin's PR

### PR DESCRIPTION
**Pull request by joos2010kj**
Finished the test cases for filter.js.

Still working on aggregate.js and bin.js. 

Questions
- I'm kind of confused on which feature needs to be tested b/c it's impossible to test it exhaustively.
- Given that we will be using the technique in a generated dataset, are the codes currently written correctly (in filter.js especially)?
- I keep getting confused with the objective.   So, we are not trying to write test cases, but we are trying to run benchmarking tests.  But what I have, following Dr. Heer's template, looks more like a test case script.

### CODE REVIEW

File: benchmarking.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+    
+        static max(item) {
+            return {
+                "type": "aggregate",
+                "fields": [item],
+                "ops": ["max"]
+            }
+        }
+    
+        static groupby(item) {
```
Comment (JunranY) [inline benchmarking.js]: groupby and ops should be able to co-exist in a same example
{
  "type": "aggregate",
  "fields": ["foo"],
  "ops": ["sum"],
  "groupby": ["bar"]
}

**Comment by joos2010kj** [comment]
@JunranY Here's the change: https://github.com/joos2010kj/voyager-microbenchmarker/pull/2/commits/7fc6329aec561cd003a0d141d6e5188bd6b5c574

Again, the main update is in just benchmark-project folder

**Comment by JunranY** [comment]
other than that, it's looking good.

**Comment by joos2010kj** [comment]
Fixed

**Pull request merged**